OnixFirebaseChat/OnixFirebaseChat/Codebase/Services/ONXFirebaseService.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ONXFirebaseService.swift
+//  OnixFirebaseChat
+//
+//  Created by Anton Dolzhenko on 02.12.16.
+//  Copyright © 2016 Onix-systems. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+import Firebase
+import FirebaseAuthUI
+import FirebaseFacebookAuthUI
+import FirebaseGoogleAuthUI
+import FirebaseTwitterAuthUI
+import TwitterKit
+
+final class ONXFirebaseService: NSObject {
+    
+    var subject:PublishSubject<FIRAuth?> = PublishSubject<FIRAuth?>()
+    private let disposeBag = DisposeBag()
+    
+    private func subscribe(){
+        just(element: FUIAuth.defaultAuthUI()?.auth)
+            //TODO: - find why bindto stops subject after subscription and try to replace .subscribe to .bindTo
+            .subscribe(onNext: { (auth) in
+                self.subject.on(.next(auth))
+            })
+            .addDisposableTo(disposeBag)
+    }
+    
+    func configure(){
```

Comment:
In some methods you have no space between closing round bracket and opening curly bracket. Please add space in every occurrence to keep code persistent.